Storage/Storage/Model/GeneralAppSettings.swift

```diff
@@ -13,36 +13,36 @@ public struct GeneralAppSettings: Codable, Equatable, GeneratedCopiable {
     /// Note that this is not accurate because this property/setting was created when we have
     /// thousands of users already.
     ///
-    public let installationDate: Date?
+    public var installationDate: Date?
```

Comment:
just wondering, is it possible for the properties to remain immutable? I don't see an easy way to support a generic `setValue` without making properties mutable though 😞

Reply:
I don't think so, `WritableKeyPath` only works for properties that are mutable. I'm generally an advocate for immutable models, but `GeneralAppSettings` is not the kind of model that gets passed around, and right now `AppSettingsStore` is the only thing that mutates it.